### PR TITLE
Add a new uptime monitor > Radio buttons: unselected radio buttons are hard to see #1404 #1412

### PR DESCRIPTION
**Pull request by Anand-Royy**
## Describe your changes
Changed the visibility of the radio buttons and made it darker.

## Issue Number
#1404 

## Please ensure all items are checked off before requesting a review:
- [x] I deployed the code locally.
- [x] I have performed a self-review of my code.
- [x] I have included the issue # in the PR.
- [x] I didn't use any hardcoded values (otherwise it will not scale, and will make it difficult to maintain consistency across the application).
- [x] I have labelled the PR correctly.
- [x] My PR is granular and targeted to one specific feature.
- [x] I took a screenshot or a video and attached this to PR if there is a UI change.

## Screenshot

WhatsApp Image 2024-12-16 at 21 17 04_f9f24705

### CODE REVIEW

File: Client/src/Components/Inputs/Radio/index.jsx [hunk 1]
```diff
@@ -42,6 +42,9 @@ const Radio = (props) => {
 						width: 16,
 						height: 16,
 						boxShadow: `inset 0 0 0 1px ${theme.palette.secondary.main}`,
+						"&:not(.Mui-checked)": {
+							boxShadow: `inset 0 0 0 1px ${theme.palette.text.primary}70`, // Use theme text color for the outline
```
Comment (ajhollid) [inline Client/src/Components/Inputs/Radio/index.jsx:46]: Does it make sense to be using the primary text color for a box shadow?  That doesn't seem semantically correct to me.

@marcelluscaio could you comment on this one?  I'll defer to your judement.

It seems to me that just removing the `color: transparent` property of the Radio button should be sufficient here :thinking:

Comment (Anand-Royy) [reply Client/src/Components/Inputs/Radio/index.jsx:46]: Hi @ajhollid , by removing the `color: transparent ` property, it makes the radio button have a  more darker outline than the checkbox. If that is fine then I'll implement it.

Comment (ajhollid) [reply Client/src/Components/Inputs/Radio/index.jsx:46]: > Hi @ajhollid , by removing the `color: transparent ` property, it makes the radio button have a more darker outline than the checkbox. If that is fine then I'll implement it.

@Anand-Royy yes I noticed that as well, that's the default appearance of the MUI radio button.

Let's see what @marcelluscaio has to say about the colors, he will be able to clarify RE using a text color here.

@gorkem-bwl should the unselected radio buttons match the color of the checkboxes? Or should they use the default color?

Comment (gorkem-bwl) [reply Client/src/Components/Inputs/Radio/index.jsx:46]: > @gorkem-bwl should the unselected radio buttons match the color of the checkboxes? Or should they use the default color?

@ajhollid yes, it should match the color of the checkboxes. 

@Anand-Royy make sure you add a screenshot here as per the PR guidelines.

image

Comment (Anand-Royy) [reply Client/src/Components/Inputs/Radio/index.jsx:46]: @gorkem-bwl yes I have added the screenshot just below that under Screenshot section. Is that the correct place to add the screenshot?

Comment (gorkem-bwl) [reply Client/src/Components/Inputs/Radio/index.jsx:46]: > @gorkem-bwl yes I have added the screenshot just below that under Screenshot section. Is that the correct place to add the screenshot?

I missed that! Thanks for adding.

Comment (Anand-Royy) [reply Client/src/Components/Inputs/Radio/index.jsx:46]: > Let's see what @marcelluscaio has to say about the colors, he will be able to clarify RE using a text color here.

Waiting for @marcelluscaio

Comment (gorkem-bwl) [reply Client/src/Components/Inputs/Radio/index.jsx:46]: Ok we can move forward if we don't hear from him today :)

**Review by ajhollid — APPROVED** [review]
I think we can make an executive decision here to merge this with the caveat that we may need to revisit that text.primary color.

**Comment by gorkem-bwl** [comment]
> I think we can make an executive decision here to merge this with the caveat that we may need to revisit that text.primary color.

Sounds good!

File: Client/src/Components/Inputs/Radio/index.jsx (hunk 1 above)
Comment (marcelluscaio) [inline Client/src/Components/Inputs/Radio/index.jsx:46]: Sorry for the delay, guys. I'm not sure about this, but it seems to have hot fixed the problem

**Pull request merged**